src/task.ts

```diff
@@ -117,7 +117,7 @@ export default class Task {
 
     const instance = await this.deploy(name, args, from, force, libs);
 
-    await this.verify(name, instance.address, args, libs);
+    await this.verify(name, instance.address, args, undefined, libs);
```

Comment:
Following the comment above:

```suggestion
    await this.verify(name, instance.address, args, libs);
```

(you can also pass an explicit `undefined` or even `this` as the last argument)